.expeditor/update_version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" lib/win32/version.rb
```

Comment:
There is no `lib/win32/version.rb` file in win32-taskscheduler. You mean `lib/win32/taskscheduler.rb`

The `win32-taskscheduler.gemspec` file also has the version in it. For chef in the gemspec we `require chef/version` and set `s.version = Chef::VERSION`. You could do something similar so we don't have to change two files.